src/routes/Rockets.jsx

```diff
@@ -1,8 +1,69 @@
-const Rockets = () => (
-  <div className="rockets-form-wrapper">
-    <div className="rockets-wrapper">
-      Rockets
-    </div>
-  </div>
-);
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+import { cancelReserveRocket, reserveRocket, getRockets } from '../components/redux/rockets/rocketsSlice';
+import styles from '../styles/Rockets.module.css';
+
+const Rockets = () => {
```

Comment:
We need to make the rockets route the default... It should load as the home page according to the requirement.

<img width="236" alt="image" src="https://github.com/Bifabig/space-travelers-hub-react-redux/assets/17850501/77c138a0-3553-4d5e-b33d-b06efaab61ee">